shared/src/main/java/uk/gov/di/authentication/shared/entity/ErrorResponse.java

```diff
@@ -43,7 +43,9 @@
     ERROR_1035(1035, "User entered invalid mfa code"),
     ERROR_1036(1036, "User entered invalid email verification code"),
     ERROR_1037(1037, "User entered invalid phone verification code"),
-    ERROR_1038(1038, "Invalid Authentication Request");
+    ERROR_1038(1038, "Invalid Authentication Request"),
+    ERROR_1039(1039, "User entered invalid password reset code"),
```

Comment:
There is already `ERROR_1021(1021, "Invalid Password reset code"),`

Reply:
Good spot

Reply:
Updated